test/inst_url_test.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "InstURL"
+Yast.import "Linuxrc"
+
+describe Yast::InstURL do
+  subject(:inst_url)  { Yast::InstURL }
+    
+  describe "#installInf2Url" do
+    let(:zypp_repo_url) { "http://opensuse.org/repo" }
+    let(:ssl_verify) { "yes" }
+
+    before do
+      Yast::InstURL.main
+      allow(Yast::Linuxrc).to receive(:InstallInf).with("ZyppRepoURL")
+        .and_return(zypp_repo_url ? zypp_repo_url.clone : nil)
```

Comment:
Why do we need to clone the string? I think it is OK to mutate strings inside methods as an implementation detail, but for API purposes we should treat them as immutable. (Ruby 3 may make them so).